### PR TITLE
Sw breastcancer conceptmodel

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

File: docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 1]
```diff
@@ -496,13 +496,84 @@ We assume family history does not affect incidence rates from LCIS to breast can
...
+   * - Treatment among women diagnosed with LCIS
+     - Proportion%
+   * - lumpectomy only
+     - 
+   * - lumpectomy + radiotherapy
+     - 
+   * - mastectomy
+     - 
+   * - no surgery
+     -
```
Comment (yongqx) [inline docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst:571]: The treatment distribution for LCIS from King (2015) is:
surveillance alone = 78%
surveillance + chemoprevention = 17%
mastectomy = 5%

File: docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 2]
```diff
@@ -496,13 +496,84 @@ We assume family history does not affect incidence rates from LCIS to breast can
...
+
+
+- :math:`i_{BC|LICS}` is the incidence of breast cancer from LCIS (baseline) based on GBD's i_BC
+- let :math:`P_{tx1}` be the proportion of people who have LCIS and treatment. 
+- :math:`1- P_{tx1}` is the poportion of people who have LCIS and no treatment
+- let :math:`i_{BC|LCIS{tx1}}` be the incidence of breast cancer from LCIS after treatment.
+- let  :math:`i_{BC|LCIS{tx0}}` be the incidence of breast cancer from LCIS among those without treatment (we can caclulate this based on the equation above and apply to scale up scenario). 
+
+
+(1) :math:`i_{BC|LCIS{tx1}}` = 488 per 100,000 PY from King 2015, Journal of Clinical Oncology
```
Comment (yongqx) [inline docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst:548]: The incidence of breast cancer for those diagnosed with LCIS and received tamoxifen alone = 383 per 100,000 PY from Fisher 1998, JNCI.

File: docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 3]
```diff
@@ -496,13 +496,84 @@ We assume family history does not affect incidence rates from LCIS to breast can
...
+(2) :math:`P_{tx1}` = 30% as baseline screening uptake x 100% as treatment uptake
+(3) :math:`i_{BC|LCIS{tx0}}` = solve for this value  
+
+.. note:: 
+   - using stand-in values
+   - verification- check to see if :math:`i_{BC|LCIS{tx0}} > i_{BC|LCIS{tx0}}`
```
[the remark this answers is not in the document]
Comment (Ninicorn) [reply docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst:554]: yess

File: docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 4]
```diff
@@ -496,13 +496,84 @@ We assume family history does not affect incidence rates from LCIS to breast can
...
+   * - lumpectomy + radiotherapy
+     - 17.6%
+   * - mastectomy
+     - 27.8%
+   * - no surgery
+     - 4.7%
+
+.. note:: 
+   - using stand-in values
+   - verification- check to see if   :math:`i_{BC|DCIS{tx0}} > i_{BC|DCIS{tx0}}`
```
[the remark this answers is not in the document]
Comment (Ninicorn) [reply docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst:533]: yes

File: docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst:548]: hmm I wonder how we can use this information, we might need to know the distribution of treatment types within our sim population. I'll add this incidence in the note.

File: docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 5]
```diff
@@ -495,6 +496,99 @@ We assume family history does not affect incidence rates from LCIS to breast can
...
+
+
+- :math:`i_{BC|DICS}` is the incidence of breast cancer from DCIS (baseline) based on GBD's i_BC
+- let :math:`P_{tx1}` be the proportion of people who have DCIS and treatment. 
+- :math:`1- P_{tx1}` is the poportion of people who have DCIS and no treatment
+- let :math:`i_{BC|DCIS{tx1}}` be the incidence of breast cancer from DCIS after treatment.
+- let  :math:`i_{BC|DCIS{tx0}}` be the incidence of breast cancer from DCIS among those without treatment (we can caclulate this based on the equation above and apply to scale up scenario). 
+
+
+(1) :math:`i_{BC|DCIS{tx1}}` = 882 (95%CI 845-921) per 100,000 person years (Mannu 2020 **BMJ**)
```
Comment (aflaxman) [inline docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst:511]: I wonder how this compares to the GBD value.  It is possible that there is not a consistent way to solve for `i_{BC|DCIS{tx0}}`.  I guess we shall see.

File: docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst [hunk 6]
```diff
@@ -495,6 +496,99 @@ We assume family history does not affect incidence rates from LCIS to breast can
...
+     - 4.7%
+
+.. note:: 
+   - using stand-in values
+   - verification- check to see if   :math:`i_{BC|DCIS{tx1}} > i_{BC|DCIS{tx0}}` 
+   - also note conclusions from Narod 2015: However, although it is accepted that,
+    for women with invasive breast cancer, prevention of inbreast
+    recurrence does not prevent death,26 this has not been
+    widely accepted for women with DCIS. Also, for women with
+    invasive cancers it is acceptedthat, intermsof survival, lumpectomy
```
Comment (aflaxman) [inline docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst:538]: copy-edit

**Pull request merged**